packages/common/src/HardhatConfig.ts

```diff
@@ -70,13 +72,30 @@ export function getHardhatConfig(
         testBlacklist,
       },
       localhost: { url: "http://127.0.0.1:9545", timeout: 1800000, testBlacklist },
+      mainnet: { chainId: 1, url: getNodeUrl("mainnet", true), accounts: { mnemonic } },
       rinkeby: { chainId: 4, url: getNodeUrl("rinkeby", true), accounts: { mnemonic } },
-      kovan: { chainId: 42, url: getNodeUrl("kovan", true), accounts: { mnemonic } },
       goerli: { chainId: 5, url: getNodeUrl("goerli", true), accounts: { mnemonic } },
-      mumbai: { chainId: 80001, url: getNodeUrl("polygon-mumbai", true), accounts: { mnemonic } },
-      matic: { chainId: 137, url: getNodeUrl("polygon-matic", true), accounts: { mnemonic } },
-      mainnet: { chainId: 1, url: getNodeUrl("mainnet", true), accounts: { mnemonic } },
+
+      kovan: { chainId: 42, url: getNodeUrl("kovan", true), accounts: { mnemonic } },
       optimism: {
+        ovm: true,
+        chainId: 10,
+        url: getNodeUrl("optimism", true),
+        accounts: { mnemonic },
+        gasPrice: 15000000,
```

Comment:
Out of curiosity, why do we need a hardcoded gas price?

Reply:
all configs I've seen do this for optimism. the hardhat hat defaults are not representative of what the mainnet is like. This is what all the tutorial/example repos have so I just copied that.

Reply:
Makes sense!